.github/workflows/scheduled_expo_build.yml

```
@@ -0,0 +1,24 @@
+name: Scheduled Expo Build
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: yarn --frozen-lockfile
```

I haven't setup node CI config before but just checking if we need to run the entire command?

```suggestion
        run: yarn install --frozen-lockfile
```

It looks like that's what the action uses: https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md?plain=1#L365

---

I _think_ that when we run just `yarn`, we are running `yarn install` 🤔 that's how we have it in `.github/workflows/ci.yml:23`. 

In any case, I think for clarity for the developer reading the action, I will add the install as you suggest 🙂 . Thank you @stefannibrasil

---

yeah, good point. It is probably what yarn does behind the scenes. For consistency and clarification, I think it's a good addition. Thank you!